README.md

```diff
@@ -73,7 +82,11 @@ This will parse the outputData.json file and update the [Issues.html](./OutputFi
 Mariner ships with a default GitHub Action that runs every 8 hours to generate a fresh issue list,
 and commit that issue list back into the GitHub repository.
 Any fork of this repository will automatically include this action,
-as it is triggered by the existance of the [action YAML file](./.github/workflows/main.yml).
+as it is triggered by the existence of the [action YAML file](./.github/workflows/main.yml).
+
+## Use Mariner Issue Collector In A Private Repo
+
+To use the demo app in a private repo, you will need to add an SSH key to Mariner Issue Collector that has read access to the private repository.
```

Comment:
I didn't even think about the use case of having Mariner look for issues in private repos, that's a neat idea! Is the only way to do that to check in a private SSH key? That seems like a significant risk.

What I wanted to clarify is the process someone wants to follow if they want to run Mariner Issue Collector privately, to protect the list of input dependencies, rather than running it publicly. If you fork this repo, you're not able to make your fork private without doing some additional configuration. That's what I'd like to capture.